test/parallel/test-fs-promises-file-handle-stat.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const common = require('../common');
+
+// The following tests validate base functionality for the fs/promises
+// FileHandle.stat method.
+
+const { open } = require('fs/promises');
+const path = require('path');
+const tmpdir = require('../common/tmpdir');
+const assert = require('assert');
+const tmpDir = tmpdir.path;
+
+tmpdir.refresh();
+common.crashOnUnhandledRejection();
+
+async function validateStat() {
+  const filePath = path.resolve(tmpDir, 'tmp-read-file.txt');
```

Comment:
`tmpDir` is used used only once here.
Can we remove variable `tmpDir` and pass `tmpdir.path` here, so that there's no confusion between similar looking variables `tmpdir` and `tmpDir`

Reply:
@trivikr 
Thank you. I fixed it.